ipaclient/plugins/dns.py

```diff
@@ -62,7 +62,7 @@ def __get_part_param(rrtype, cmd, part, output_kw, default=None):
 def prompt_parts(rrtype, cmd, mod_dnsvalue=None):
     mod_parts = None
     if mod_dnsvalue is not None:
-        name = record_name_format % rrtype.lower()
+        name = unicode(record_name_format % rrtype.lower())
```

Comment:
You should use `decode` method on rrtype to prevent:
_BytesWarning: str() on a bytes instance_
And you should also remove unicode() call.

Reply:
Thanks, done.